src/community/release/ext-webp.xml

```diff
@@ -0,0 +1,17 @@
+<assembly>
+    <id>webp-plugin</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>release/target/dependency</directory>
+            <outputDirectory></outputDirectory>
+            <includes>
+                <include>gs-webp*.jar</include>
+                <include>webp-imageio-0.2.2.jar</include>
```

Comment:
Maybe this would make the assembly less prone to error in case the webp-imageio library gets updated.

```suggestion
                <include>webp-imageio-*.jar</include>
```